web/.env.example

```
@@ -48,7 +48,13 @@ PGHOST=
 PGPORT=
 PGUSER=
 
-# Optional Stack Auth create-credit gating. Leave unset to use active VM limits only.
+# Stack Auth create-credit gating. The free plan consumes one team-scoped Stack Auth
+# item credit per successful create. Configure the Stack Auth free/include-by-default
+# product to grant 20 of this item. Set this to "none" to disable create credits.
+CMUX_VM_PLAN_FREE_CREATE_CREDIT_ITEM_ID=cmux-vm-create-credit
+CMUX_VM_PLAN_FREE_CREATE_CREDIT_COST=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMUX_VM_PLAN_FREE_CREATE_CREDIT_COST key should go before the CMUX_VM_PLAN_FREE_CREATE_CREDIT_ITEM_ID key

(UnorderedKey)